LogicalTypes.md

```diff
@@ -255,6 +255,18 @@ The primitive type is a 2-byte fixed length binary.
 
 The sort order for `FLOAT16` is signed (with special handling of NANs and signed zeros); it uses the same [logic](https://github.com/apache/parquet-format#sort-order) as `FLOAT` and `DOUBLE`.
 
+### FIXED_SIZE_LIST
+
+The `FIXED_SIZE_LIST` annotation represents a fixed-size list of elements
+of a fixed-width data type. It must annotate an N-byte fixed length binary
```

Comment:
Why couldn't we support variable width data types here? Effectively this type would act like a repeated element, but with the repetition levels not needing to be specified?

Reply:
My assumption was `fixed_len_byte_array` will give us better performance than `binary` but we then require constant bit width. Are you saying we could relax this condition, use `binary` and cover non-fixed-width types?

Reply:
I suppose `binary` vs `fixed_len_byte_array` wouldn't affect performance that much. Changed.
I wonder if it would make sense to also propose a non-nested LIST type?